src/wasi_libc.zig

```diff
@@ -10,13 +10,17 @@ pub const CrtFile = enum {
     crt1_reactor_o,
     crt1_command_o,
     libc_a,
+    libdl_a,
```

Comment:
this strikes me as strange... wasi doesn't support dynamic linking, so what in the world is this library?

Reply:
```c
weak void *dlopen(const char *name, int flags)
{
	error = "dlopen not implemented";
	return NULL;
}

weak void *dlsym(void *library, const char *name)
{
	error = "dlsym not implemented";
	return NULL;
}
```

oh.

ok, well, it won't be very difficult to implement this in zig :roll_eyes:

Reply:
It is just another one of those "emulated" WASI libraries that do nothing so existing software can compile. For whatever reason it doesn't follow the usual naming pattern.

FWIW, these all strike me as silly and I wouldn't be opposed to removing them, but that'd probably be separate work.